Enable Windows

jeongseok-meta opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

jeongseok-meta committed:
Enable Windows

jeongseok-meta commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14344072258](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14344072258). Examine the logs at this URL for more detail.</sub>

jeongseok-meta committed:
CONDA_BLD_PATH: C:\\bld\\

jeongseok-meta commented:
@conda-forge-admin, please rerender

jeongseok-meta committed:
MINIFORGE_HOME: C:\\Miniforge\\

jeongseok-meta commented:
@conda-forge-admin, please rerender

jeongseok-meta commented:
@conda-forge-admin, please rerender

jeongseok-meta committed:
MINIFORGE_HOME: C:\Miniforge

jeongseok-meta commented:
@conda-forge-admin, please rerender

jeongseok-meta committed:
Create bld.bat

jeongseok-meta committed:
Remove 7.0 from TORCH_CUDA_ARCH_LIST

jeongseok-meta committed:
Update bld.bat

jeongseok-meta committed:
Update bld.bat

jeongseok-meta committed:
Update bld.bat

jeongseok-meta commented:
@conda-forge-admin, please rerender

jeongseok-meta committed:
vs2022

jeongseok-meta commented:
@conda-forge-admin, please rerender

jeongseok-meta committed:
Pin CUDA package versions

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15711747237](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15711747237). Examine the logs at this URL for more detail.</sub>

jeongseok-meta committed:
Update conda_build_config.yaml

jeongseok-meta commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I failed to even lint the recipe, probably because of a conda-smithy bug :cry:. This likely indicates a problem in your `meta.yaml`, though. To get a traceback to help figure out what's going on, install conda-smithy and run `conda smithy recipe-lint --conda-forge .` from the recipe directory. You can also examine the [workflow logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15711827648) for more detail.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15711827648](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15711827648). Examine the logs at this URL for more detail.</sub>

jeongseok-meta committed:
Update meta.yaml

jeongseok-meta committed:
multiple cuda versions

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15712586139](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15712586139). Examine the logs at this URL for more detail.</sub>

jeongseok-meta committed:
Update bld.bat

jeongseok-meta committed:
Update conda_build_config.yaml

jeongseok-meta commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I failed to even lint the recipe, probably because of a conda-smithy bug :cry:. This likely indicates a problem in your `meta.yaml`, though. To get a traceback to help figure out what's going on, install conda-smithy and run `conda smithy recipe-lint --conda-forge .` from the recipe directory. You can also examine the [workflow logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15713856244) for more detail.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15713856244](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15713856244). Examine the logs at this URL for more detail.</sub>

jeongseok-meta committed:
Update conda_build_config.yaml

jeongseok-meta commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19801533578](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19801533578). Examine the logs at this URL for more detail.</sub>

jeongseok-meta committed:
Update bld.bat

jeongseok-meta committed:
Update bld.bat

jeongseok-meta committed:
Update conda_build_config.yaml

jeongseok-meta committed:
Update bld.bat

jeongseok-meta committed:
Update bld.bat

jeongseok-meta commented:
@conda-forge-admin, please rerender

jeongseok-meta commented:
@conda-forge-admin, please rerender

jeongseok-meta committed:
Update bld.bat

jeongseok-meta committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.11.29.08.28.23

Other tools:
- conda-build 25.11.0
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

jeongseok-meta committed:
Enable cuda-toolkit on Windows

jeongseok-meta committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.11.29.08.28.23

Other tools:
- conda-build 25.11.0
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

jeongseok-meta committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.11.29.08.28.23

Other tools:
- conda-build 25.11.0
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

jeongseok-meta committed:
Use VS2019 for Windows build to satisfy CUDA 12.9 requirement

jeongseok-meta committed:
Fix syntax error in bld.bat

jeongseok-meta committed:
Use separate header file for cooperative_groups patch

jeongseok-meta committed:
Add debug prints to bld.bat

jeongseok-meta committed:
Replace unicode characters with ASCII in bld.bat

jeongseok-meta committed:
Replace unicode characters with ASCII in bld.bat

jeongseok-meta committed:
Simplify bld.bat to debug environment issues
